fxa-auth-db-server/docs/API.md

```diff
@@ -69,13 +69,13 @@ The following datatypes are used throughout this document:
     * deleteSessionToken        : `DEL /sessionToken/:id`
     * createSessionToken        : `PUT /sessionToken/:id`
     * updateSessionToken        : `POST /sessionToken/:id/update`
-    * verifyToken               : `POST /token/:tokenVerificationId/verify`
```

Comment:
(thanks for reminding me though, I should have called that out explicitly in my list of "the auth server must..." things in the PR description)